icu4c/source/common/unicode/urename.h

```diff
@@ -203,6 +204,7 @@
 #define res_read U_ICU_ENTRY_POINT_RENAME(res_read)
 #define res_unload U_ICU_ENTRY_POINT_RENAME(res_unload)
 #define u_UCharsToChars U_ICU_ENTRY_POINT_RENAME(u_UCharsToChars)
+#define u_asciiToUpper U_ICU_ENTRY_POINT_RENAME(u_asciiToUpper)
```

Comment:
This one seems a bit odd... The function isn't public and seems to be an internal only API, yet the name starts with `u_` (instead of `uprv_`) and the API is also marked with the macro `U_CAPI` instead of `U_INTERNAL`.

Reply:
if not public it should be U_INTERNAL — or even statiic?

Reply:
Well, U_INTERNAL is defined to be U_CAPI so there is no implementation difference. For documentation purposes this could be updated, but that would be better as part of either the API status update or the guard conditionals update (both of which I am doing).

Reply:
this function is called only from `i18n/currunit.cpp` and implemented in `common/ustring.cpp`, so U_INTERNAL is the right choice.

Reply:
Fixed this in https://github.com/unicode-org/icu/pull/1081